patroni/postgresql/__init__.py

```diff
@@ -407,6 +407,12 @@ def start(self, timeout=None, task=None, block_callbacks=False, role=None):
         self._pending_restart = False
 
         configuration = self.config.effective_configuration
+        if "unix_socket_directories" in configuration:
+            for d in configuration["unix_socket_directories"].split(","):
+                if not d.startswith('/'):
```

Comment:
1. this is not compatible with windows
2. we have to `d.strip()`
2. `os.path.join()` is actually smart enough to do it unconditionally:
```python
>>> import os
>>> os.path.join('/foo', '/bar')
'/bar'
```

Reply:
fixed